Use QStandardPaths instead of hard coded paths

dcaliste opened the pull request:
Close #20 

Also adjust some icon sizes, since large version are not existing for up and down.

Olf0 reviewed (approved):
@dcaliste, wow, thanks a lot!

Side not / question: Have you addressed all hardcoded `/home/nemo` paths, or shall I still check if there are some remaining to be addressed?

dcaliste commented:
Normally, I've addressed all cases. Not all old paths are now treated 1:1 with the QStandardPaths approach. I'm thinking for instance to the database location, which was previously in `~/.config/`, but I've now moved it to `~/.config/flowplayer/flowplayer` (the name redundancy is due to the way APP_CONFIG path is created with a style like `org/app`, and here the organisation is declared as `flowplayer` as for the application name). This latter path is compatible with Sailjail for instance.
